### PR TITLE
fix: add proper labels when registering strain wrapper

### LINKED ISSUE #3368: Strain components' naming issue
**Describe the issue**
The strain component xz is actually named with the index "yz" and vice versa.

**Proposed cleanup**
Correct the names of the strain components: "yz" to "xz" and "xz" to "yz"

**Additional context**

### PR DESCRIPTION
**Pull request by ryar9534**
Simply added the same explicit labelling that is done for stress to strain.

### CODE REVIEW

File: src/coreComponents/physicsSolvers/solidMechanics/SolidMechanicsLagrangianFEM.cpp [hunk 1]
```diff
@@ -158,7 +160,7 @@ void SolidMechanicsLagrangianFEM::registerDataOnMesh( Group & meshBodies )
     {
       setConstitutiveNamesCallSuper( subRegion );
 
-      subRegion.registerField< solidMechanics::strain >( getName() ).reference().resizeDimension< 1 >( 6 );
+      subRegion.registerField< solidMechanics::strain >( getName() ).setDimLabels( 1, voightLabels ).reference().resizeDimension< 1 >( 6 );
```
Comment (rrsettgast) [inline src/coreComponents/physicsSolvers/solidMechanics/SolidMechanicsLagrangianFEM.cpp:163]: what about stress?

**Review by rrsettgast — APPROVED** [review]
I don't understand what was generating the labels prior to this specification? Do you know what was happening?

**Comment by CusiniM** [comment]
> I don't understand what was generating the labels prior to this specification? Do you know what was happening?

if you don't specify labels vtk assigns them but the ordering of a rank 2 tensor in Voigt notation used by vtk (or I guess by Paraview more precisely) is different from ours.

File: src/coreComponents/physicsSolvers/solidMechanics/SolidMechanicsLagrangianFEM.cpp (hunk 1 above)
Comment (CusiniM) [reply src/coreComponents/physicsSolvers/solidMechanics/SolidMechanicsLagrangianFEM.cpp:163]: For stress we also specify our own labels [here](https://github.com/GEOS-DEV/GEOS/blob/c225416b8ca7efcc08e13503f96dbc08e3c925f3/src/coreComponents/constitutive/solid/SolidBase.cpp#L37)

**Review by CusiniM — APPROVED** [review]

**Comment by sytuannguyen** [comment]
I tested this PR, it fixed the issue #3368, thank you!

**Pull request merged**